.gitignore

```diff
@@ -29,5 +29,6 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+/config/credentials.yml.enc
```

Comment:
why add the .yml here?

Reply:
previously used to help debug the credentials yml file. Extraneous now